<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingRanking</class>
 <widget class="QWidget" name="SettingRanking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="vLayoutSettingRanking">
   <item>
    <layout class="QHBoxLayout" name="hLayoutTop">
     <item>
      <widget class="QPushButton" name="pbSelectClass">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>직업 선택</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbInfo">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vLayoutSettingData"/>
   </item>
   <item>
    <widget class="QPushButton" name="pbShowMore">
     <property name="text">
      <string>더보기</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
